vendor/vendor.json

```diff
@@ -43,12 +43,10 @@
 			"versionExact": "v21.1.0"
 		},
 		{
-			"checksumSHA1": "a/mbRxz450lwbacAUuG77DpKKJU=",
-			"path": "github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-04-01/containerinstance",
+			"checksumSHA1": "eB6b4vGKEbAND0MUWfKLTn6H7uw=",
+			"path": "github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-06-01/containerinstance",
 			"revision": "6d20bdbae88c06c36d72eb512295417693bfdf4e",
-			"revisionTime": "2018-09-28T00:20:07Z",
-			"version": "v21.1.0",
-			"versionExact": "v21.1.0"
+			"revisionTime": "2018-09-27T22:44:43Z"
```

Comment:
You missed `version` here. Please use this command:
`govendor fetch github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-06-01/containerinstance@v21.1.0`